### DISCUSSION TITLE
Auto close single and double quotes

### DISCUSSION THREAD

**Discussion by AJ-The-Wolf** [brackets-cont/brackets#280] [Q&A]
Hi all,
I'm not able to set Brackets to close ' and " automatically, just like it does with ( ), [ ] or { }. Is this feature available? And, if yes, how do I enable it?
Thanks in advance!

**Comment by charlypa** [comment]
I think its supported please check the attached gif. Please share  what is missing for you
![Brackets_oU71qUEo76](https://github.com/brackets-cont/brackets/assets/1525027/9678d173-830f-47a7-80c9-7fb9c18150c4)

**Comment by AJ-The-Wolf** [comment]
Well I have nothing to share actually. Simply, when I type ' or " (in any file) Brackets doesn't close them automatically. ( [ and { work correctly instead. Brackets 2.2.1 on Windows without extensions.

**Comment by AJ-The-Wolf** [comment]
After some tests I found the problem actually is with plain text files. Brackets works correctly with all file types but "simple text" ones. I mean, when I edit a text file (.txt or any other extension that can be considered as "plain text") Brackets auto-closes ( , [ , and { , but not ' and ". Are there any settings to fix this?

**Comment by charlypa** [comment]
@AJ-The-Wolf 
Apologies for the late reply. In text files this feature is not supported

We've updated our editor to Phoenix Code, launched just yesterday. It now supports Windows, Mac, and Linux moving beyond the older Brackets x86 binaries.
You can download Phoenix Code at https://phcode.io/.

**Comment by MrDrone-ru** [comment]
> Phoenix Code

Is it a cross-platform replacing of Brackets? Will it have that [feature](https://github.com/brackets-cont/brackets/issues/319)?

**Comment by charlypa** [comment]
@MrDrone-ru 
Phoenix code has this feature and it is cross-platform.
